arangod/Scheduler/SupervisedScheduler.cpp

```diff
@@ -214,27 +229,32 @@ bool SupervisedScheduler::queue(RequestLane lane, std::function<void()> handler,
   static thread_local uint64_t lastSubmitTime_ns;
 
   // use memory order release to make sure, pushed item is visible
-  uint64_t jobsSubmitted = _jobsSubmitted.fetch_add(1, std::memory_order_release);
-  uint64_t approxQueueLength = jobsSubmitted - _jobsDone;
+  uint64_t const jobsDone = _jobsDone.load(std::memory_order_acquire);
+  uint64_t const jobsSubmitted = _jobsSubmitted.fetch_add(1, std::memory_order_relaxed);
```

Comment:
This increment has to happen _before_ the job is pushed onto the queue. (and of course we have to decrement it again if the push fails)